doc/Learn/Tutorials/Http/HowTo-EndpointOptions.md

```diff
@@ -0,0 +1,221 @@
+---
+uid: Learn.Tutorials.Http.HowToEndpointOptions
+---
+
+# How-To: Configure `HttpClient` with Custom Endpoint Options
+
+It's often necessary to include an API key alongside requests to a web API. This can be done by adding a header to the request. The steps below will show you how to easily specify custom options, such as an access token, when adding an endpoint. You can then configure the associated `HttpClient` from these options.
+
+## Pre-requisites
+
+* An [environment](xref:Uno.GetStarted) set up for developing Uno Platform applications
+
+* The solution was created using the template wizard or `dotnet new unoapp`. See [Overview](xref:Overview.Extensions)
+
+* Basic conceptual understanding of accessing web resources using HTTP requests
+
+* Knowledge of how to [register an endpoint for HTTP requests](xref:Learn.Tutorials.Http.HowToHttp)
```

Comment:
@nickrandolph Let me know what you think about this. As our docs get more specific and helpful for less-trivial scenarios, it may make sense to make the intended audience more obvious (within reason). Bonus points if we can link to existing material.

I'd be interested in your take on this idea.